addons/web/static/src/core/overlay/overlay_container.scss

```diff
@@ -1,4 +1,4 @@
 .o-overlay-item {
     position: fixed;
-    z-index: $zindex-modal;
+    z-index: $zindex-modal + 5;
```

Comment:
Can you explain how to reproduce the issue since that change was not in the initial PR?

Reply:
```
isActive: ["desktop"],
trigger: "a.dropdown-item[id*='user_ids'] span",
content: markup(_t("Select an <b>assignee</b> from the menu")),
position: "right" 
```

Try making adding this step(in on-boarding manual tour) and there would be a problem. Now as we have removed the step from on boarding we may not need change but this a generic issue cause it may occur else where. What do you think?

Reply:
We could also remove this If you want.

Reply:
No, we can let your fix since there is an issue :)

Reply:
Hello @odoo/rd-framework-js ,

Can we have your opinion?

We do that change because the dropdown of a many2many could be above than the tooltip message for the onboarding tour.

for instance: 
![image](https://github.com/user-attachments/assets/dbb9e8b6-bd55-4f5c-bc8a-33fd6c33e8b1)

I hope it will be good for you. 🙂 

Have a nice day! 🙂

Reply:
Thanks @BastienFafchamps 🙂